entry/src/main/resources/base/element/string.json

```diff
@@ -0,0 +1,44 @@
+{
+  "string": [
+    {
+      "name": "entry_MainAbility",
+      "value": "entry_MainAbility"
+    },
+    {
+      "name": "mainability_description",
+      "value": "Java_Empty Ability"
+    },
+    {
+      "name": "mainability_HelloWorld",
```

Comment:
remove unnecessary strings, other string values also please add value related to library